### DISCUSSION TITLE
[NEWS] New AzerothCore ecosystem

### DISCUSSION THREAD

**Discussion by Yehonal** [azerothcore/azerothcore-wotlk#3919] [General]
Hi AC folks! How's going? :)

Recently someone has noticed that there're some "moving" on azerothcore organization, new repositories [popped up](https://github.com/azerothcore/noderouter) [here](https://github.com/azerothcore/keira3) and [there](https://github.com/azerothcore/universal-pwa), some ["unknown devs"](https://github.com/azerothcore/acore-pwa/commits/master) appeared, and we've been working on some well-organized projects ([DBC](https://github.com/azerothcore/azerothcore-wotlk/projects/13), [PWA](https://github.com/azerothcore/acore-pwa/projects/1)).

So, I'm going to explain now what's happening under the hood.

**To improve the experience with the AzerothCore project we decided to create an entire ecosystem of officially supported projects that will allow you to have a full set of tools to create your own digital idea, whatever it will be!**

We've been working on several software during these years for our jobs and other personal projects and we decided to move on azerothcore most of such resources to be reused internally (for the azerothcore project) and distributed (for general-purpose).

To list some of them:

- **[ACore-CMS](https://github.com/azerothcore/acore-cms):** PHP website based on Wordpress with an official AC plugin that will allow you to create accounts, sell items, create a blog, a forum, and much more!

- **[ACore-node-server](https://github.com/azerothcore/acore-node-server):** Based on "universal-node boilerplate" it's a modular NodeJS platform that will create a layer of GraphQL & REST APIs above azerothcore-wotlk 

- **[ACore-PWA](https://github.com/azerothcore/acore-pwa):** Based on "universal-pwa boilerplate" it's a modular progressive web app platform written in ReactJS that can consume ACore-node-server and ACore-CMS APIs. It can replace ACore-CMS if you have the possibility to host such multi-service architecture.

- **[ACore-Client](https://github.com/azerothcore/acore-client):** a ReactJS client that will be directly connected to the azerothcore-wotlk opcodes implementing the game chat system in a fancy web app. It will be also used to create e2e tests for the server opcodes.

- **[AS-Launcher:](https://github.com/azerothshard/get-as-launcher)** a freeware software that will allow you to download modern graphic patches, creating a list of realmlists to easily switch between servers, adjusting client configurations before launching the game, and much more!

- **ACore-Framework:** a subset of Azerothcore-wotlk libraries to create a reusable framework to build your own cpp server project.

That's the model that will come up:

![image](https://user-images.githubusercontent.com/147092/81286589-95f04380-9061-11ea-9925-ad7d4c2f7e25.png)

With such ecosystem you will be able to create your personal MMO community with all the tools you need but you can also use one of these projects for any kind of project you want to build up. From a simple chat to a full-featured website.

**In our case, we will use all software above to create a full-featured azerothcore demo** with a new website where you will be able to create accounts, try the azerothcore features on a demo server, download open-source or premium modules and tools and "play" with our web client. In the future we will also create a point-based leaderboard with all your contributions (in a retro-active way) letting you access our premium modules for free! So, keep up the good work!

We're using an AGILE-Scrum framework to coordinate most of the projects I've listed above and to complete some critical/needed stuff on azerothcore-wotlk, but we're few volunteers right now! **Given that, to speed up this process, we need the support of you all!** 
Please contact me on Discord if you want to join one of our AGILE-Scrum workflows. If you're not able to help us with the development **please support us** via a bountysource donation: https://salt.bountysource.com/checkout/amount?team=azerothcore. We will place such resources for the development of this amazing project and to improve the quality of our azerothcore main project itself.

Tell us your opinion about it and, Stay tuned!

A special thanks to @breizh24, @Nek97, @ShinDarth, @helias, and all other amazing people that are helping me with this epic job.

**Comment by BRNKR** [comment]
Amazing News!
